core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -131,6 +131,7 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     GainFreeBuildings("Gain a free [buildingName] [cityFilter]", UniqueTarget.Global, UniqueTarget.Triggerable),
     BuildingMaintenance("[relativeAmount]% maintenance cost for buildings [cityFilter]", UniqueTarget.Global, UniqueTarget.FollowerBelief),
     RemoveBuilding("Remove [buildingFilter] [cityFilter]", UniqueTarget.Global, UniqueTarget.Triggerable),
+    SellBuilding("Sell [buildingFilter] [cityFilter]", UniqueTarget.Global, UniqueTarget.Triggerable),
```

Comment:
Looking at this again, these should be phrased "Sell [buildingFilter] buildings [cityFilter]" - that way we can say e.g. "Sell [Culture] buildings [in all cities]" and it looks correct

Reply:
Done